builds/mac_os_x/CS/Helpers.make

```diff
@@ -262,24 +243,11 @@ $(PS_FW_FLAG): $(FULL_FW_FLAG) $(UPG_FW_FLAG)
 	touch $(PS_FW_FLAG)
 darwin_pseudo_fw_clean:
 
-build_dbs_: $(MSG_DB) $(HELP_DB) $(META_DB)
-build_dbs_clean:
-	rm -f $(MSG_DB) $(HELP_DB) $(META_DB)
-build_dbs_%:
```

Comment:
Maybe I'm missing something, but shouldn't this remain for `$(MSG_DB)`?

Reply:
I'm not sure this file should even exist.